docs/_static/mxnet-theme/navbar.html

```diff
@@ -7,47 +7,39 @@ <h1 id="logo-wrap">
       <nav id="main-nav" class='nav-bar'>
         <a class="main-nav-link" href="{{url_root}}install/index.html">Install</a>
 
-        <span id="dropdown-menu-position-anchor">
-          <a href="#" class="main-nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="true">Gluon <span class="caret"></span></a>
-          <ul id="package-dropdown-menu" class="dropdown-menu navbar-menu">
-            <li><a class="main-nav-link" href="{{url_root}}gluon/index.html">About</a></li>
-            <li><a class="main-nav-link" href="http://gluon.mxnet.io">Tutorials</a></li>
```

Comment:
Why remove gluon ? No easy way for users to get to the gluon book now ?

Reply:
You can get to the Gluon book from the tutorials page. Plus @thomelane will be moving that link on the tutorials page above the fold, so it will be easier to see.